rpcserver.go

```diff
@@ -1865,8 +1870,7 @@ func createRPCChannel(r *rpcServer, graph *channeldb.ChannelGraph,
 	var chanID uint64
 	chanID, _ = graph.ChannelID(&chanPoint)
 
-	// Next, we'll determine whether we should add this channel to
-	// our list depending on the type of channels requested to us.
+	// Next, we'll determine whether the channel is public or not.
```

Comment:
minor nit: make part of the original commit that changed this.

Reply:
fixed